core/src/main/java/nu/yona/server/subscriptions/service/BuddyService.java

```diff
@@ -302,9 +373,12 @@ void sendBuddyConnectResponseMessage(UUID senderUserID, UUID senderUserAnonymize
 
 	private void disconnectBuddy(UserAnonymizedDTO userAnonymized, UUID userAnonymizedID)
```

Comment:
Done: ``disconnectBuddyIfConnected``, and also renamed ``userAnonymized`` into ``buddyUserAnonymized``.